test/system/250-systemd.bats

```diff
@@ -479,4 +479,41 @@ $name stderr" "logs work with passthrough"
     run_podman generate --help
     is "$output" ".*\[DEPRECATED\] Generate systemd units"
 }
+
+@test "podman network reload on firewall-cmd --reload" {
+    setup_firewalld_services
+
+    systemctl daemon-reload
+
+    reload_service="podman-firewalld-reload.service"
+    systemctl start $reload_service
+    systemctl is-active $reload_service
+
+    restart_service="podman-firewalld-restart.service"
+    systemctl start $restart_service
+    systemctl is-active $restart_service
+
+    cname="testctr"
+    run_podman run -d --rm --name $cname fedora:latest sleep 10d
+
+    # reload firewalld
+    firewall-cmd --reload
+
+    # ensure the rules are present
+    fout=$(run firewall-cmd --zone=trusted --list-all | grep "sources")
+    assert "$fout" != "  sources: " # non-empty
+
+    # restart firewalld service
+    systemctl restart firewalld.service
+
+    # ensure the rules are still present
+    fout=$(run firewall-cmd --zone=trusted --list-all | grep "sources")
+    assert "$fout" != "  sources: " # non-empty
```

Comment:
This is just asking for flakes, there is no guarantee that by the time you check the network reload command was already run.
Also it would make much more sense to actually test the rules are restored by ding connectivity tests like done in the networking tests.